doc/api/modules.md

```diff
@@ -159,6 +159,7 @@ require(X) from module at path Y
 3. If X begins with './' or '/' or '../'
    a. LOAD_AS_FILE(Y + X)
    b. LOAD_AS_DIRECTORY(Y + X)
+   c. THROW "not found"
 4. LOAD_NODE_MODULES(X, dirname(Y))
```

Comment:
alternatively, could this be prefixed with “else”?

Reply:
I thought of that but noticed that other steps here prefer throwing in the if instead.

Reply:
I agree it's consistent with the other blocks.